core/modules/system/system.module

```diff
@@ -3596,6 +3596,23 @@ function system_archiver_info() {
 }
 
 /**
+ * Implements hook_admin_bar_alerts().
+ */
+function system_admin_bar_alerts() {
+  module_load_include('inc', 'system', 'system.admin');
+  if (!system_status(TRUE)) {
```

Comment:
Let's also remove the backdrop_set_message() in this same PR. The admin bar alert should be enough.